src/com/facebook/buck/jvm/groovy/GroovycStep.java

```diff
@@ -61,8 +69,8 @@
 
   @Override
   public int execute(ExecutionContext context) throws IOException, InterruptedException {
-
-    ProcessBuilder processBuilder = new ProcessBuilder(createCommand());
+    ImmutableList<String> command = createCommand();
+    ProcessBuilder processBuilder = new ProcessBuilder(command);
```

Comment:
I'm going to undo the spurious part in this PR, but I think switching to ProcessExecutor or equivalent should probably be a separate commit.